src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -966,15 +1011,24 @@ export async function isApplicationUp(projectID: string, handler: any): Promise<
             updateDetailedAppStatus(projectID, containerInfo.ip, port, path);
         }
 
+        const isHttps: boolean = await getProjectWWWProtocolMap(projectID);
```

Comment:
Do we need the `getProjectWWWProtocolMap()` function? Shouldn't `projectInfo` contain the `isHttps` property? If we don't need that function we don't need the `projectWWWProtocolMap` either.